ibm/data_source_ibm_is_dedicated_host_disk.go

```diff
@@ -0,0 +1,234 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+)
+
+func dataSourceIbmIsDedicatedHostDisk() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIbmIsDedicatedHostDiskRead,
+
+		Schema: map[string]*schema.Schema{
+			"dedicatedhost": &schema.Schema{
```

Comment:
change to dedicated_host we followed same reference in other related resources/datasource